### PR TITLE
feat: add in docs for generic storage provider

### PR DESCRIPTION
**Pull request by SorsOps**
This is an initial draft to show consumers how to use the new generic storage provider available in the v129 release of the figma tokens plugin

### CODE REVIEW

File: pages/sync/generic-storage.en.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+---
+
+import Callout from 'nextra-theme-docs/callout'
+
+
+![](/generic-storage-sync.png)
+
+
+Sometimes you need more control over how your tokens are stored, either to circumvent direct git solutions like Github, Gitlab, etc, to keep control within your infrastructure and not rely on third party storage, or to adopt more fine grained control regarding access and modification of tokens by your users, in this case, you can create your own storage endpoint and hook it into the Figma Tokens plugin.
```
Comment (six7) [inline pages/sync/generic-storage.en.mdx:11]:
```suggestion
Sometimes you need more control over how your tokens are stored, either to circumvent direct git solutions like GitHub, Gitlab, etc, to keep control within your infrastructure and not rely on third party storage, or to adopt more fine grained control regarding access and modification of tokens by your users, in this case, you can create your own storage endpoint and hook it into the Figma Tokens plugin.
```

super small nit

File: pages/sync/generic-storage.en.mdx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+### 4. Push or pull changes
+
+Tokens should be automatically updated whenever you make changes, however you might need to manually sync to your endpoint occasionally if multiple users are writing to the same endpoint.
+
+### 5. How to use tokens stored in Generic Storage in development?
+
+This storage provider stores the raw json representing your tokens, as such you will need to be responsible in disseminating the useable resolved tokens to your downstream consumers.
+
+To help you, the following package could be useful: [Token Transformer](https://www.npmjs.com/package/token-transformer). You or your dev can install it by running `npm install token-transformer`. You can then pull through the tokens json file from your endpoint, extract the `values` and `$themes` properties on the response and handle the resolution locally, or implement this functionality directly into your API for your consumers
```
Comment (six7) [inline pages/sync/generic-storage.en.mdx:67]:
```suggestion
To help you, the following package could be useful: [Token Transformer](https://www.npmjs.com/package/token-transformer). You or your dev can install it by running `npm install token-transformer`. You can then pull through the tokens json file from your endpoint, extract the `values` and `$themes` properties on the response and handle the resolution locally, or implement this functionality directly into your API for your consumers.
```

File: pages/sync/generic-storage.en.mdx [hunk 3]
```diff
@@ -0,0 +1,67 @@
+# Generic Versioned Storage Sync
```
Comment (six7) [inline pages/sync/generic-storage.en.mdx:1]: can you also make sure to add the .pt-br version of this? can stay english, but at least those users won't get a 404 then (plus translators will be able to see which files are not yet translated and possibly offer a PR

**Review by six7 — CHANGES_REQUESTED** [review]
Only some smaller nits, and the pt-br stuff, otherwise let's get this merged 👍

File: pages/sync/generic-storage.en.mdx (hunk 3 above)
Comment (SorsOps) [reply pages/sync/generic-storage.en.mdx:1]: Will do !

**Review by six7 — APPROVED** [review]

**Pull request merged**